R/geom_phylopic.R

```diff
@@ -76,10 +81,10 @@ geom_phylopic <- function(mapping = NULL, data = NULL,
     position = position,
     show.legend = show.legend,
     inherit.aes = inherit.aes,
-    params = list(
+    params = c(
       na.rm = na.rm,
       remove_background = remove_background,
-      ...
+      dots
```

Comment:
should we make this `args = dots`? Or similar?

Reply:
dots is already a named list, so here we are just concatenating it with the other named items.

Reply:
Yes, it was just more for clarity, but it is also fine!